03_backend_developement/01_HTTP/05_centralized_webservice/shell_functions/fetch_comments.sh

```diff
@@ -0,0 +1,2 @@
+touch shell_functions/fetch_comments.result
+curl http://jsonplaceholder.typicode.com/comments > shell_functions/fetch_comments.result
```

Comment:
Just to be sure, these are not to be written by students but to be used directly?
If yes, maybe you could explain it in the exercise explanation on `sparta_courses`?

Reply:
Nope ! They are given, and they are already seen in the previous exercises